cluster-autoscaler/core/scale_down_test.go

```diff
@@ -113,6 +124,7 @@ func TestFindUnneededNodes(t *testing.T) {
 	context := AutoscalingContext{
 		AutoscalingOptions: AutoscalingOptions{
 			ScaleDownUtilizationThreshold: 0.35,
+			ExpendablePodsPriorityCutoff:  10,
```

Comment:
Why is it 10? Doesn't that make every pod in test expendable (I think default is 0)?

Reply:
I have added comment to flag and test case. Pods with null priority are always non expendable.

Reply:
In order to be backward compatible, I actually think pods with no priority class or with 0 priority should be expendable. Pods with negative priority should be non-expendable by default.

Reply:
Didn't you mean the opposite?
expendable - below 0 (or  below ExpendablePodsPriorityCutoff if defined)?
non-expendable - null priority or greater or equal to 0 (ExpendablePodsPriorityCutoff if defined)
Before priority preemption - priority was always null so all pods are non expendable.
I think I should newer check priority class. Right?

Reply:
Yes, reading my comment again, it should be the order way. I just edited it. Thanks and sorry for the confusion.